sage-ios/SAGE/SAGE/UIFont.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  UIFont.swift
+//  SAGE
+//
+//  Created by Sameera Vemulapalli on 10/8/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import UIKit
+
+extension UIFont {
+    // metatext: 12
+    // normal: 14
+    // title: 17 (regular)
+    // 20-45 light
+    // above 45 thin
+    // semibold instead of bold
+
+    @nonobjc static let metaFont = UIFont.systemFontOfSize(12)
```

Comment:
but we wanted it to be san francisco for everything right? thats why we imported the font in the first place. so that itll be SF for iOS 8 too